packages/lit-element/src/env.d.ts

```diff
@@ -30,10 +30,9 @@ interface ShadyDOM {
   wrap: (node: Node) => Node;
 }
 
-interface Window {
+interface LitExtraGlobals {
```

Comment:
If this interface extends `Window` you won't need the casts to `unknown`.

I can't quite get that to work though: https://www.typescriptlang.org/play?#code/FAFwngDgpgBA4gGwPYCMCGCYF4YHUCWAdgCZIDuMAZDONEgGYwDmy6CAKgBb4DOA3MGBEQUAE700AY1gES5AKIAPEKLQ8YUZVBLrEqDDADewGDHpIkALhiEArgFsUYgQF9BxKJIRpRsAG4+MGREpGTWsqFKKmoCwAAUwXIUanghCsqqPACUAHTmSAJAA

Reply:
Yeah, I'm having trouble with this too. If I just say `extends Window` and remove the `as unknowns`, then I get this:

```
src/polyfill-support.ts(97,24): error TS2352: Conversion of type 'Window & typeof globalThis' to type 'LitExtraGlobals' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

But I can't seem to write whatever it _should_ extend properly:

```ts
interface LitExtraGlobals extends (Window & typeof globalThis) { ... }
```

```
src/env.d.ts(33,35): error TS2499: An interface can only extend an identifier/qualified-name with optional type arguments.
```

Reply:
Oh, this works:

```ts
const extraGlobals = window as LitExtraGlobals & typeof globalThis;
```

It's longer but it doesn't have `unknown`. WDYT?

Reply:
I just pushed some changes that extend `Window` here and make a new type to add the `globalThis`-ness into it so that `window` can be casted directly, PTAL.